Java/Dice/src/domain/Dices.java

```diff
@@ -0,0 +1,16 @@
+package domain;
+
+import java.util.ArrayList;
+
+public class Dices {
+    ArrayList<Dice> dices = new ArrayList<>();
+    public Dices(int number){
+        for(int i=0; i<number; i++){
+               dices.add(new Dice());
```

Comment:
`IntStream`을 써보는것이 어떨까요?
```suggestion
       IntStream.rangeClosed(0, number).forEach(i->dices.add(new Dice()));
```